build.bat

```diff
@@ -92,6 +93,7 @@ rem parse the command line options
   if "%1" == "weasel" set build_weasel=1
   if "%1" == "installer" set build_installer=1
   if "%1" == "arm64" set build_arm64=1
+  if "%1" == "testing_cast" set build_test_cast=1

```

Comment:
Make it clear and easy to type: "appcast"
Optionally also change the variable to build_appcast